src/layers/MapboxLayer.js

```diff
@@ -6,6 +6,21 @@ import OLLayer from 'ol/layer/Layer';
 import GeoJSON from 'ol/format/GeoJSON';
 import Layer from './Layer';
 
+const getCopyrightFromSources = mbMap => {
+  let copyrights = [];
+  const regex = /<[^>]*>[^>]*<\/[^>]*>/g;
+  // Trick from Mapbox AttributionControl to know if the source is used.
+  const { sourceCaches } = mbMap.style;
+  Object.entries(sourceCaches).forEach(([, sourceCache]) => {
+    if (sourceCache.used) {
```

Comment:
In this example, `sourceCache.used` is false. Is this intended?

Reply:
We display the source's copyright only if the source is used by the style.